monailabel/transform/pre.py

```diff
@@ -60,3 +93,129 @@ def __call__(self, data):
             label[label > 0] = self.value
             d[key].array = label
         return d
+
+
+class SuppressStderr:
+    """Suppress stderr. Useful as OpenCV (and dependencies) can produce a lot of output."""
+
+    def __enter__(self):
+        self.errnull_file = open(os.devnull, "w")
+        self.old_stderr_fileno_undup = sys.stderr.fileno()
+        self.old_stderr_fileno = os.dup(sys.stderr.fileno())
+        self.old_stderr = sys.stderr
+        os.dup2(self.errnull_file.fileno(), self.old_stderr_fileno_undup)
+        sys.stderr = self.errnull_file
+        return self
+
+    def __exit__(self, *_):
+        sys.stderr = self.old_stderr
+        os.dup2(self.old_stderr_fileno, self.old_stderr_fileno_undup)
+        os.close(self.old_stderr_fileno)
+        self.errnull_file.close()
+
+
+class LoadVideoFrame(Transform):
```

Comment:
May I know why you make it as a transform?
Maybe you can use the MONAI `VideoFileDataset` direclty:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/video_dataset.py#L141

Thanks.